packages/playground-examples/copy/ko/JavaScript/JavaScript Essentials/Hello World.ts

```diff
@@ -0,0 +1,34 @@
+//// { order: 0, compiler: { target: 1 } }
+
+// TypeScript 플레이그라운드에 오신 것을 환영합니다.
+// 이 웹 사이트는 TypeScript를 웹 브라우저에서 실행하는 것과 거의 비슷합니다.
+
+// 플레이그라운드에서는 이러한 프로젝트를 공유하는 것을
+// 간단한 일로 만들었기 때문에, TypeScript로 아이디어를
+// 쉽고 안전하게 실험할 수 있습니다. 다른 사람에게 프로젝트를
+// 공유할 때 필요한 것은, 이 페이지의 URL이 전부입니다.
+
+const hello = "Hello";
+
+// 오른쪽에서 TypeScript 컴파일러의 결과를 볼 수
+// 있습니다. : 이것은 브라우저나 서버는 물론, 어디에서나
+// 실제로 실행할 수 있는 바닐라 JavaScript입니다.
```

Comment:
```suggestion
// 실제로 실행할 수 있는 vanilla JavaScript입니다.
```
`React`처럼 `vanill`도 원문이 나을 것 같아요.

Reply:
앗 알겠습니다~ 그럼 이 관련사항도 용어사전에 적어두겠습니다. 🤗🤗